tests/Collection/CollectionFunctionalTest.php

```diff
@@ -104,6 +106,40 @@ public function testGetNamespace()
         $this->assertEquals($this->getNamespace(), $this->collection->getNamespace());
     }
 
+    public function testAggregateWithinTransaction()
```

Comment:
I decided to add tests for `aggregate()` in response to https://github.com/mongodb/mongo-php-library/issues/562#issuecomment-404792939. Best I can tell, there are no problems with `aggregate()`, which jives with the underlying libmongoc bug.